### PR TITLE
Better error reporting.

### PR DESCRIPTION
**Pull request by Maggie-M**

### CODE REVIEW

File: bokeh/sphinxext/bokeh_model.py [hunk 1]
```diff
@@ -72,20 +73,26 @@ class BokehModelDirective(Directive):
...
+
         model_path = self.arguments[0]
         module_name, model_name = model_path.rsplit(".", 1)
 
         try:
             module = importlib.import_module(module_name)
         except ImportError:
-            pass
+            raise SphinxError("Unable to generate reference docs for %s, couldn't import module '%s'" %
+            (model_path, module_name))
```
Comment (mattpap) [inline bokeh/sphinxext/bokeh_model.py]: This line (and two more below) should be indented.

**Comment by bryevdv** [comment]
LGTM thanks @Maggie-M and @mattpap

**Comment by bryevdv** [comment]
I do think this change actually breaks on some docs currently (as it should) so I am going to investigate and possibly make a separate PR to merge into this one with any necessary fixes to get docs building clean again before merging this.

**Comment by bryevdv** [comment]
OK, actually the docs build without errors, so I will merge this now, and make a separate PR for cleanup of docs and examples warnings later.

**Pull request merged**